docs/docs/messaging-and-voice-channels.mdx

```diff
@@ -44,24 +34,24 @@ Learn how to make your assistant available on:
 * [Custom Connectors](./connectors/custom-connectors.mdx)
 
 
-## Testing Channels on Your Local Machine with Ngrok
+## Testing Channels on Your Local Machine
 
-You can use [ngrok](https://ngrok.com/) to create a connection to your local
-computer that is publicly available on the internet.
-You don't need this when running Rasa on a server because, you can set up a domain
-name to point to that server's IP address, or use the IP address itself.
+If you're running Rasa X Local mode or a Rasa Open Source server on `localhost`, 
+most external channels won't be able to find your server URL, since `localhost` is not open to the internet. 
+
+To make a port on your local machine publicly available on the internet, 
+you can use [ngrok](https://ngrok.com/).
 
 After installing ngrok, run:
 
 ```bash
 ngrok http 5005; rasa run
 ```
 
-Your webhook address will look like the following:
-
-* `https://yyyyyy.ngrok.io/webhooks/<CHANNEL>/webhook`, e.g.
-
-* `https://yyyyyy.ngrok.io/webhooks/facebook/webhook`
+When you follow the instructions to make your assistant available on a channel, use the ngrok URL:
+wherever the instructions say to use `https://<host>:<port>/webhooks/<CHANNEL>/webhook`,
+use `https://yyyyyy.ngrok.io/webhooks/<CHANNEL>/webhook`, replacing `yyyyyy` with the randomly generated
```

Comment:
```suggestion
use `https://<url>.ngrok.io/webhooks/<CHANNEL>/webhook`, replacing `<url>` with the randomly generated
```

Reply:
Mm that chunk itself is not a url though 🤔